libs/pages/application/src/lib/ui/page-settings-advanced/page-settings-advanced.tsx

```diff
@@ -0,0 +1,186 @@
+                <Tooltip content={key}>
+                  <div>{key}</div>
+                </Tooltip>
+              ),
+              className: 'font-medium vis',
+            },
+            {
+              content: (
+                <>
+                  <Tooltip
+                    content={
+                      (props.defaultAdvancedSettings &&
+                        props.defaultAdvancedSettings[key as keyof ApplicationAdvancedSettings]?.toString()) ||
+                      ''
+                    }
+                  >
+                    <div className="inline whitespace-nowrap overflow-hidden text-ellipsis">
+                      {props.defaultAdvancedSettings &&
+                        props.defaultAdvancedSettings[key as keyof ApplicationAdvancedSettings]?.toString()}
+                    </div>
+                  </Tooltip>
+                  <CopyToClipboard
+                    className="ml-2 text-text-300 invisible group-hover:visible"
+                    content={
+                      (props.defaultAdvancedSettings &&
+                        props.defaultAdvancedSettings[key as keyof ApplicationAdvancedSettings]?.toString()) ||
+                      ''
+                    }
+                  />
+                </>
+              ),
+              className: 'group',
+            },
+            {
+              className: '!p-1',
+              content: (
+                <Controller
+                  name={key}
+                  control={control}
+                  rules={{
+                    required: 'Please enter a value.',
+                  }}
+                  defaultValue=""
+                  render={({ field, fieldState: { error } }) => (
+                    <InputTextSmall
+                      className="shrink-0 grow flex-1"
+                      data-testid="value"
+                      name={field.name}
+                      onChange={field.onChange}
+                      value={field.value}
+                      error={error?.message}
+                      errorMessagePosition="left"
+                      transparentBorderOnValid={true}
+                      label={field.name}
+                    />
+                  )}
+                />
+              ),
+            },
+          ],
+        }
+      })
+    : []
+
+  table = [...table, ...tableBody]
+
+  return (
+    <div className="flex flex-col justify-between w-full">
+      <div className="p-8 ">
+        <div className="flex justify-between mb-4">
+          <div>
+            <h1 className="h5 text-text-700 mb-2">Advanced Settings</h1>
+            <p className="text-sm text-text-500 max-w-content-with-navigation-left">
+              Settings are injected at the build and run time of your application and thus any change on this section
+              will be applied on the next manual/automatic deploy.
+            </p>
+          </div>
+        </div>
+
+        <InputToggle
+          className="mb-4"
```

Comment:
I think you can add the props `small` for it because actually, the toggle is huge vs the rest of the page, and added more bottom marge like `mb-6` 

```        
   <InputToggle
      className="mb-6"
      small
      dataTestId="show-overriden-only-toggle"
      value={showOverriddenOnly}
      onChange={toggleShowOverriddenOnly}
      title="Show only overridden settings"
   />
```

<img width="1070" alt="Capture d’écran 2022-08-30 à 12 30 32" src="https://user-images.githubusercontent.com/533928/187414762-437350b2-305f-48ab-ae59-369bfb72c479.png">